Products/ZenEvents/zentrap.py

```diff
@@ -545,17 +545,21 @@ def _value_from_dateandtime(self, value):
                 mon, day, hour, mins, secs, dsecs, utc_dir, utc_hours, utc_mins)
 
     def _convert_value(self, value):
-        if not isinstance(value, basestring):
-            return value
-        try:
-            value.decode('utf8')
-            return value
-        except UnicodeDecodeError:
-            # Try converting to a date
-            decoded = self._value_from_dateandtime(value)
-            if not decoded:
-                decoded = 'BASE64:' + base64.b64encode(value)
-            return decoded
+        # Check if values is OID (SNMP object id)
+        if isinstance(value, tuple) and len(value) > 2:
```

Comment:
This change reverts the current _convert_value() function from the one we are using in 5.x to the older version from the 4.2.5-RPS repo.  Are we reverting intentionally?